tests/cpp-tests/Classes/FileUtilsTest/FileUtilsTest.cpp

```diff
@@ -139,15 +139,15 @@ void TestSearchPath::onEnter()
 
     // Save old resource root path
     std::string oldDefaultRootPath = sharedFileUtils->getDefaultResourceRootPath();
-    sharedFileUtils->setDefaultResourceRootPath("extensions");
+    sharedFileUtils->setDefaultResourceRootPath(oldDefaultRootPath + "extensions");
```

Comment:
This line fix the issue. The new resource root path must be associated with the old one, then plus "extension".